### ISSUE TITLE
Feature request:  include `rgl` output automatically.

### ISSUE THREAD

**Issue by dmurdoch** [tidyverse/reprex#404]
It would be nice if `rgl` output was automatically included in `reprex()` output.  This should be easy to do because `knitr` has supported `rgl` output for a while now.  All that is needed is to run `rgl::setupKnitr(autoprint = TRUE)` and snapshots of output will be produced.  

To set this automatically, `reprex()` could run the following code:

```
setHook(packageEvent("rgl", "onLoad"),
        function(...) rgl::setupKnitr(autoprint = TRUE))
```

I would be happy to submit a PR to do this if you give me a hint where it should happen.  Here's what output would look like with this addition (though of course the`setHook()` call won't be displayed if `reprex()` runs it internally):
``` r
setHook(packageEvent("rgl", "onLoad"),
        function(...) rgl::setupKnitr(autoprint = TRUE))
library(rgl)
plot3d(1:10, 1:10, 1:10)
```

![](https://i.imgur.com/8K31bUM.png)

<sup>Created on 2022-01-23 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>

For anyone reading this and running `reprex` without this facility, you don't really need to do it as a hook, it's enough to run the line in the code you're reproducing, e.g.

```
rgl::setupKnitr(autoprint = TRUE))
library(rgl)
plot3d(1:10, 1:10, 1:10)
```

**Comment by jennybc** [comment]
The current plan is to probably handle this inside rgl itself. See the PR for discussion.

**Comment by dmurdoch** [comment]
Done now, included in `rgl` 0.108.21 (which is only available on Github at the moment).
